internal/core/application/operator_service.go

```diff
@@ -148,15 +148,19 @@ func (o *operatorService) DepositMarket(
 		}
 
 		nextAccountIndex := latestAccountIndex + 1
-		if _, err := o.marketRepository.GetOrCreateMarket(ctx, nextAccountIndex); err != nil {
+		fee := int64(config.GetFloat(config.DefaultFeeKey) * 100)
```

Comment:
Also this should become a field of `operatorService` and we pass down when we create it